static/bidder-info/bidgency.yaml

```diff
@@ -0,0 +1,10 @@
+aliasOf: aso
+endpoint: "https://srv.bidgx.com/pbs/bidder?zid={{.ZoneID}}"
+maintainer:
+  email: "aso@bidgency.com"
```

Comment:
sent email for verification. Please reply with received.

Reply:
They replied, please check your inbox.